CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -1193,11 +1193,6 @@ void __CFInitialize(void) {
         CFNumberGetTypeID();		// NB: This does other work
 
         __CFCharacterSetInitialize();
-        
-#if DEPLOYMENT_TARGET_WINDOWS
-        __CFWindowsNamedPipeInitialize();
-#endif
-        
```

Comment:
Did we remove this completely?

Reply:
Nearly completely ... had two straggling references remaining which I just cleaned up :-).